spec/lib/integrations/openai/processor_service_spec.rb

```diff
@@ -97,5 +97,193 @@
         expect(result).to eq('This is a reply from openai.')
       end
     end
+
+    context 'when event name is label_suggestion with labels' do
```

Comment:
spec for `label_Suggestions`  should be in enterprise spec ? -> think its already covered

Reply:
@sojan-official Removed.